### ISSUE TITLE
Leaflet appears to have no screen reader support

### ISSUE THREAD

**Issue by mehgcap** [Leaflet/Leaflet#6511]
**Steps to reproduce**

* Implement Leaflet on a page, load up a screen reader, and try to interact with the controls. The samples I've seen (tested with VoiceOver on iOS 12 and NVDA on Firefox 65) don't work with these screen readers at all.
* Try to arrow, tab, or swipe on the map. The screen reader will speak unlabeled images in place of map data, or will speak nothing except links and zoom control buttons. Attributions, Leaflet, and other links are identified correctly, but the actual map data is not at all accessible.

**Expected behavior**

The screen reader should have some way of interacting with the map data. There's no perfect way I know of to do this, but a few models come to mind. That's another discussion, though.

**Current behavior**

Leaflet isn't rendering map data in a way that is even somewhat accessible.

**Environment**
- Leaflet version: 1.4.0
- Browser (with version): Firefox 65 on Windows 10, Safari on iOS 12 (latest public beta)
- OS/Platform (with version): Windows 10 (v1809), iOS 12.2 beta 1
- Screen Readers: NVDA 2018.4.1, VoiceOver on iOS 12.2 beta

**Additional context**

I realize that asking for maps to be accessible might seem odd. Why, after all, would screen reader user need maps, when websites could offer text-based alternatives? Isn't a map rather useless if one can't see it?

* Websites rely on maps to let users pinpoint a location. A web-based study on area transportation options, for instance, wants the user to mark their current location on a map so the study knows where participants are.
* Websites rarely bother to go the extra mile in accessibility, so the likelihood is high that they will not make an alternative UI for map data
* POIs, important intersections, and road layouts are all valuable information for which there should not be accessibility barriers

**Minimal example reproducing the issue**

Missing images and lack of data, though not unlabeled images, can be experienced by using a screen reader to interact with the samples on Leaflet's own [Quick Start Guide](https://leafletjs.com/examples/quick-start/).

**Comment by IvanSanchez** [comment]
> the actual map data is not at all accessible

Alas, that's not a Leaflet problem, that's a data provider problem.

If I recall correctly, the OSM folks did some work related to textual representations of map tiles in the past. As of today, I'm not aware of any map data provider which does such a thing in a reliable way.

Otherwise, closing as duplicate of https://github.com/Leaflet/Leaflet/issues/3210
